src/Entity/PropertyWrapper/BackedEnumWrapper.php

```diff
@@ -30,8 +30,11 @@ public function convertToRawValue(mixed $value): mixed
 		if ($value === null) return null;
 		$type = array_key_first($this->propertyMetadata->types);
 		assert($value instanceof $type);
-		assert($value instanceof BackedEnum);
-		return $value->value;
+		if ($value instanceof BackedEnum) {
+			return $value->value;
+		}
```

Comment:
What's the motivation here? Why should we allow not BackendEnum? If you don't need the conversion, is there any reason to use BackedEnumWrapper?

Reply:
irrelevant